internal/super/flix.go

```diff
@@ -256,6 +261,13 @@ func generateFlixCmd(
 
 	ctx := context.Background()
 
+	// merge deployed contracts and core contracts
+	for name, addresses := range coreContracts {
+		if _, exists := depContracts[name]; !exists {
```

Comment:
What if they are different contracts but the same name? This won't work. You could use the source string as identifiers. There is a function for that somewhere.

Reply:
How do string imports work for contracts that import multiple contracts with same name? Also can you point out an example of core contracts that have the same name?

Reply:
Or are you saying there would be an issue if the user has deployed a contract that uses the same name as a core contract? in that case if the user already has a contract name "FungibleToken" then that contract would be used instead of the core contract. Which I think is the correct behavior.

Reply:
Actually since FLIX only deals with transactions and scripts. Is it possible for them to use String imports and import a contract same name and different addresses?

Reply:
> How do string imports work for contracts that import multiple contracts with same name? Also can you point out an example of core contracts that have the same name?

You can't import two contracts with the same name. They have to be named different in the `flow.json`. Since you're using the name to filter it's possible you could have a name in your `flow.json` matching the name from core contracts, which is why using an identifier like `testnet://0x123.Hi` might be a better ID to filter on. 

> Or are you saying there would be an issue if the user has deployed a contract that uses the same name as a core contract? in that case if the user already has a contract name "FungibleToken" then that contract would be used instead of the core contract. Which I think is the correct behavior.

There is no guarantee that because someone names it `FungibleToken` in their `flow.json` its actually the `FungibleToken` contract. 

Question: Do you even need to pull the core contracts, because wouldn't someone making a FLIX with imports have the core contract listed in their `flow.json`?

> Actually since FLIX only deals with transactions and scripts. Is it possible for them to use String imports and import a contract same name and different addresses?

Can't import two with the same name.

Reply:
ok, I think all is good then. I wanted to import core contracts to make it easier to generate flix. If there is an issue with multiple contracts with same name I'll look into using an identifier. I don't think the case will come up. I rather keep flix simple at this stage.

Reply:
We don't really have this pattern in the CLI where contracts are handled behind the scenes without any visibility to the developer. It seems like if you want to have the core contract handled for flix, it should also exist in your `flow.json`. What if on running flix it recognized missing core contracts and prompted if you want to add each to your project with a yes/no prompt then called Dependency Manager behind the scenes to automatically add them to a project that required them for flix?

Reply:
An error instead of the interactive (although nice) would be simpler and could tell them to install missing dependencies with Dependency Manager so they are in the `flow.json`